release-notes/aws/v9.0.5.md

```diff
@@ -0,0 +1,17 @@
+# :zap: Giant Swarm Release v9.0.5 for AWS :zap:
+
+This release fixes a problem in user values migration logic for apps, and fixing release templates to support Helm 3.
```

Comment:
```suggestion
This release fixes a bug causing the NGINX IC app to be removed from a cluster.

It also modifies release templates to support the coming upgrade to Helm 3.
```

Reply:
I wanted to make the value of the bug fix clearer. Also to communicate the coming Helm 3 upgrade. WDYT? @tomahawk28 @rossf7 ?

Same applies to 9.3.1